frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/subscribe-to-ocs.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { nav } from '../../../integration-tests-cypress/views/nav';
+import { primaryButton } from '../../../integration-tests-cypress/views/form';
+import { installSucess } from '../views/olm';
+
+describe('Subscribe to OCS Operator from OperatorHub', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    nav.sidenav.clickNavLink(['Operators', 'OperatorHub']);
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Searches and finds OCS Operator', () => {
+    cy.byTestID('search-operatorhub').type('Openshift Container Storage');
+    cy.byTestID('ocs-operator-redhat-operators-openshift-marketplace').click({ force: true });
+  });
+
+  it('Subscribes to OCS Operator', () => {
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
```

Comment:
Noticing a lot of `.click({ force: true })`, we should really root cause why we need to bypass Cypress click checking in these cases.  What error messages are you seeing?

Reply:
```Timed out retrying: cy.click() failed because this element is not visible:

<a class="pf-c-button pf-m-primary co-catalog-page__overlay-action" data-test-id="operator-install-btn" href="/operatorhub/subscribe?pkg=ocs-operator&amp;catalog=redhat-operators&amp;catalogNamespace=openshift-marketplace&amp;targetNamespace=">Install</a>

This element <a.pf-c-button.pf-m-primary.co-catalog-page__overlay-action> is not visible because its parent <div#modal-container.pf-c-backdrop__open> has CSS property: overflow: hidden and an effective width and height of: 1000 x 0 pixels.```
Is it due to the absolute positioning of the modal?
This is the only place it's required the other one is a mistake. Removing it.

Reply:
Is it due to the absolute positioning of the modal?
This is the only place it's required the other one is a mistake. Removing it.

Reply:
I think we'll need to investigate these in general, this is not the only place `.click({ force: true })` has been required.